### PR TITLE
Add Pandas sum codebyte

### PR DESCRIPTION
**Pull request by SamanthaZerman**
Added a codebyte to the bottom on Pandas sum method. Please make sure my description makes sense.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [ x] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/pandas/concepts/groupby/terms/sum/sum.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ CatalogContent:
 
 The **`.sum()`** method produces a new `Series` or [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) with aggregate sums for the groups in a [`GroupBy`](https://www.codecademy.com/resources/docs/pandas/groupby) object.
 
-## Syntax
+## Syntax for Pandas .sum() Method
```
Comment (caupolicandiaz) [inline content/pandas/concepts/groupby/terms/sum/sum.md:17]:
```suggestion
## Syntax 
```

Comment (caupolicandiaz) [reply content/pandas/concepts/groupby/terms/sum/sum.md:17]: We generally don't specify the method in the header

File: content/pandas/concepts/groupby/terms/sum/sum.md [hunk 2]
```diff
@@ -57,3 +57,20 @@ This example produces the following output:
 1   B   13.0
 2   C   12.0
 ```
+
+### Try It Yourself
```
Comment (caupolicandiaz) [inline content/pandas/concepts/groupby/terms/sum/sum.md:61]:
```suggestion
## Codebyte Example
```

Comment (caupolicandiaz) [reply content/pandas/concepts/groupby/terms/sum/sum.md:61]: I believe "try it yourself" is the tag line used on another site. We generally stick to "Codebyte Example"

File: content/pandas/concepts/groupby/terms/sum/sum.md [hunk 3]
```diff
@@ -57,3 +57,20 @@ This example produces the following output:
...
+
+Use the Codebytes editor below to produce a new Series using Pandas .sum() method.
```
Comment (caupolicandiaz) [inline content/pandas/concepts/groupby/terms/sum/sum.md:63]:
```suggestion
Use the Codebytes editor below to produce a new Series with the `.sum()` method.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@SamanthaZerman I just added a few comments. Let me know if you have any questions. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@SamanthaZerman LGTM

File: content/pandas/concepts/groupby/terms/sum/sum.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ CatalogContent:
 
 The **`.sum()`** method produces a new `Series` or [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) with aggregate sums for the groups in a [`GroupBy`](https://www.codecademy.com/resources/docs/pandas/groupby) object.
 
-## Syntax
+## Syntax
```
Comment (SSwiniarski) [inline content/pandas/concepts/groupby/terms/sum/sum.md:17]:
```suggestion
## Syntax
```

**Review by SSwiniarski — APPROVED** [review]
@SamanthaZerman Looks good to me.

**Pull request merged**